src/monitor/monitor.c

```diff
@@ -1895,6 +1779,9 @@ static void monitor_restart_service(struct mt_svc *svc)
     }
 }
 
+/* from nscd.c */
+errno_t sss_nscd_parse_conf(const char *conf_path);
+
 static void check_nscd(void)
```

Comment:
Hi,

wouldn't it be more consistent to move `check_nscd()` into `nscd.c` as well?

bye,
Sumit

Reply:
Done.

Reply:
Thank you.